trustprofile/trustprofile.php

```diff
@@ -11,7 +11,16 @@
 
 namespace TrustProfile\WordPress;
 
+namespace WebwinkelKeur\WordPress;
+
+
+use Valued\WordPress\PhpCompatibilityCheck;
 require __DIR__ . '/common/autoload.php';
-require __DIR__ . '/src/Plugin.php';
 
+if (!PhpCompatibilityCheck::isCompatible('trustprofile')) {
```

Comment:
I sent hours dealing with this one, but this is the only scenario that works as intended.
Initially, I did
```
if (PhpCompatibilityCheck::isCompatible('trustprofile')) {
    Plugin::getInstance()->init();
}
```
However, the plugin was still installed when the version was lower than 7.0.0 and it throws an error.
The current solution, when the plugin is < 7 it does not throw an error, and it shows the plugin in installed plugins but it also shows the error `admin_notice`.